BolajiOlajide/rmw-bot
# Setup Barebones flask app

#### What Does This PR Do?
Sets up a barebones flask application with an index route that returns a string.

#### Description Of Task To Be Completed
Set up a barebones flask application with an index route that returns a string.

#### Any Background Context You Want To Provide?
N/A

#### How can this be manually tested?
- Start the application with the command `python main.py`
- navigate to the route `localhost:5000` with your API client
- you should get a response identical to the screenshot below

#### What are the relevant github issues (Link)?
[Issue #3](https://github.com/BolajiOlajide/rmw-bot/issues/3)

#### Screenshot(s)
<img width="1108" alt="screen shot 2018-06-12 at 7 00 54 pm" src="https://user-images.githubusercontent.com/25608335/41308122-f90bc3f8-6e72-11e8-86cd-3e5ab4b3feb1.png">
